### ISSUE TITLE
Error code correction

### ISSUE THREAD

**Issue by xr0927** [mert-acar/mri-reconstruction#2]
In evaluate.py:
        pred = back_format(output['image'].cpu().detach().numpy())
        grnd = back_format(batch['full'].cpu().detach().numpy())

model.load_state_dict(state_dict)--------->model.load_state_dict({k.replace('module.',''):v for k,v in state_dict.items()})

after this line:    model.load_state_dict({k.replace('module.',''):v for k,v in state_dict.items()})
add:                    model.cuda()
